### ISSUE TITLE
Error when buliding debian package

### ISSUE THREAD

**Issue by zdyang** [shadowsocks/libQtShadowsocks#132]
OS: Debian GNU/Linux 9.1 (stretch)
GCC: 6.3.0 

libQtShawdowsocks: 1.11.0(release)

I used "dpkg-buildpackage -uc -us -b" to build a debian package and encountered following error:

(Skip some messages)

```
Install the project...
/usr/bin/cmake -P cmake_install.cmake
-- Install configuration: "None"
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/lib/libQtShadowsocks.so
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/QtShadowsocks
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/address.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/addresstester.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/chacha.cpp
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/chacha.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/cipher.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/common.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/controller.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/encryptor.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/encryptorprivate.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/export.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/httpproxy.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/profile.h
-- Up-to-date: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/QtShadowsocks
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/rc4.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/socketstream.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/tcprelay.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/tcpserver.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/include/QtShadowsocks/udprelay.h
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/lib/pkgconfig/QtShadowsocks.pc
-- Installing: /home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/bin/shadowsocks-libqss
-- Set runtime path of "/home/zidong/Downloads/libQtShadowsocks-1.11.0/debian/tmp/usr/bin/shadowsocks-libqss" to ""
make[1]: Leaving directory '/home/zidong/Downloads/libQtShadowsocks-1.11.0/obj-i686-linux-gnu'
   dh_install -O--buildsystem=cmake -O--parallel
dh_install: Cannot find (any matches for) "usr/include/qtshadowsocks/*" (tried in "." and "debian/tmp")
dh_install: libqtshadowsocks-dev missing files: usr/include/qtshadowsocks/*
dh_install: missing files, aborting
debian/rules:6: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
```

I have tried google this problem and also looked up at the repo existing issues but failed to fix it.

Can you help me? Thanks.

**Comment by zdyang** [comment]
I found that this problem is caused by __Capitalization__.

The header file directory in my system is "/usr/include/QtShawsock

**Comment by librehat** [comment]
If you check the install file in the repository, you can see it is `QtShadowsocks/*`, so I'm not sure why you had that issue

**Comment by zdyang** [comment]
The issue occured in released version 1.11.0.

https://github.com/shadowsocks/libQtShadowsocks/blob/v1.11.0/debian/libqtshadowsocks-dev.install

**Comment by librehat** [comment]
Ah thank you! Right I fixed it after the release..
